service/src/main/java/greencity/service/ubs/UBSManagementEmployeeServiceImpl.java

```diff
@@ -289,6 +289,25 @@ public void deactivateEmployee(Long id) {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @Transactional
+    public void activateEmployee(Long id) {
+        Employee employee = employeeRepository.findById(id)
+            .orElseThrow(() -> new NotFoundException(ErrorMessage.EMPLOYEE_NOT_FOUND + id));
+        if (employee.getEmployeeStatus().equals(EmployeeStatus.INACTIVE)) {
+            employee.setEmployeeStatus(EmployeeStatus.ACTIVE);
+            try {
+                userRemoteClient.activateEmployee(employee.getUuid());
+            } catch (HystrixRuntimeException e) {
+                throw new BadRequestException("Employee with current uuid doesn't exist: " + employee.getUuid());
```

Comment:
Do we have `HystrixRuntimeException`  only if `Employee with current uuid doesn't exist`?

Reply:
yes